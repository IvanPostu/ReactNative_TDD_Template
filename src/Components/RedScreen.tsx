import React from 'react';
import {StyleSheet, View} from 'react-native';

export default function RedScreen() {
  return <View style={styles.root} />;
}

const styles = StyleSheet.create({
  root: {
    height: 200,
    backgroundColor: 'red',
  },
});
