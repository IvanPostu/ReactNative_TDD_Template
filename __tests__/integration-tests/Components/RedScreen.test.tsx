import React from 'react';
import RedScreen from '@/Components/RedScreen';
import {render} from '@testing-library/react-native';

describe('RedScreen component test', () => {
  test('Renders correctly (take snapshot)', () => {
    const tree = render(<RedScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
