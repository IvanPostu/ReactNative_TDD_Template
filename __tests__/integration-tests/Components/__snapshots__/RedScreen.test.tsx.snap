// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RedScreen component test Renders correctly (take snapshot) 1`] = `
<View
  style={
    Object {
      "backgroundColor": "red",
      "height": 200,
    }
  }
/>
`;
